util/atlasresponse/atlas_response.go

```diff
@@ -1,4 +1,4 @@
-package atlasResponse
+package atlasresponse
 
```

Comment:
define the structure as below
// response.go
type APIResponse struct {
    HttpStatusCode  int         `json:”code,omitempty”`
    HttpError string      `json:”errorMessage,omitempty”`
    Response    interface{} `json:”response,omitempty"`
}